class.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    yield @list
+  end
+end
```

Comment:
- [ ] In this activity, we actually expect you to differently create your own `each` method from scratch instead of using the specified inbuilt array method. It should be similar to the example below:

```ruby
class MyList

[...]

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

[...]

end
```

Reply:
This change has been done well